commit 6b9d90a8973d75268174915d42f7b52f524dc2ff
Author: Jeroen Demeyer <jdemeyer@cage.ugent.be>
Date:   Sat Feb 25 22:52:12 2017 +0100

    Call get_interact_value inside "with out" block
    
    This way, exceptions raised by get_interact_value() will be shown in
    the output area

diff --git a/ipywidgets/widgets/interaction.py b/ipywidgets/widgets/interaction.py
index 0f8b737..92d9a44 100644
--- a/ipywidgets/widgets/interaction.py
+++ b/ipywidgets/widgets/interaction.py
@@ -208,12 +208,12 @@ class interactive(VBox):
         if self.manual:
             self.manual_button.disabled = True
         try:
-            for widget in self.kwargs_widgets:
-                value = widget.get_interact_value()
-                self.kwargs[widget._kwarg] = value
             with self.out:
                 if self.clear_output:
                     clear_output(wait=True)
+                for widget in self.kwargs_widgets:
+                    value = widget.get_interact_value()
+                    self.kwargs[widget._kwarg] = value
                 self.result = self.f(**self.kwargs)
                 if self.auto_display and self.result is not None:
                     display(self.result)
